.travis.yml

```diff
@@ -6,11 +6,17 @@ node_js:
 
 matrix:
   fast_finish: true
+  include:
+    - node_js: 7 # Version used to deploy to npm registry
+      env: BUILD_ENV=production
```

Comment:
This only adds one extra job to the build (which still runs all its jobs concurrently, so build time does not increase) to make sure what we deploy or will is actually correct.